hoomd/hpmc/ComputeFreeVolumeGPU.cuh

```diff
@@ -274,6 +274,11 @@ __global__ void gpu_hpmc_free_volume_kernel(unsigned int n_sample,
         Scalar yrand = hoomd::detail::generate_canonical<Scalar>(rng);
         Scalar zrand = hoomd::detail::generate_canonical<Scalar>(rng);
 
+        if (this->m_sysdef->getNDimensions() == 2)
```

Comment:
```suggestion
        if (dim == 2)
```
`m_sydef` is not available in the GPU kernel. The caller provides the number of dimensions in the `dim` argument:
https://github.com/glotzerlab/hoomd-blue/blob/73f186e074373f62008179d7b859750c66250108/hoomd/hpmc/ComputeFreeVolumeGPU.h#L250-L278